recipes-kernel/linux/linux-raspberrypi_4.19.bb

```diff
@@ -0,0 +1,17 @@
+LINUX_VERSION ?= "4.19.44"
+LINUX_RPI_BRANCH ?= "rpi-4.19.y"
+
+SRCREV = "b45a32b47b76cb64225d19841c20ee85ea97874d"
+SRC_URI = " \
+    git://github.com/raspberrypi/linux.git;protocol=git;branch=${LINUX_RPI_BRANCH} \
+    "
+
+# linux-raspberrypi.inc adds file(s) to SRC_URI: SRC_URI += "file://rpi-kernel-misc.cfg"
+# so we need to add a path to the file(s) to FILESEXTRAPATHS so it can find it
+YOCTOROOT = "${@os.path.abspath(os.path.join("${TOPDIR}", os.pardir))}"
```

Comment:
I would prefer not adding these files from here but copying what we need from a repo with a working 4.19 version to our own `files` directory.

The two main problems:
- it's brittle, we had a new version that's not explicitly supported at the location where the file lie
- it relies on the relative paths between layers

Reply:
We can also try to advocate for fixing this bug on meta-rasbperrypi:thud directly, in addition to this patch.

Reply:
actually, 4.19 does not require any additional patches or files (see the original recipe 8c9076e).
yes, the relative path can be potentially an issue.

Reply:
Then what happens if you omit these two lines?

Reply:
an error saying that `rpi-kernel-misc.cfg` cannot be found.
Anyway, please review, the latest version, it actually overrides SRC_URI so no need to search for  `rpi-kernel-misc.cfg`  which is not required in this case.